src/Compilers/VisualBasic/Portable/VisualBasicCompilationOptions.vb

```diff
@@ -253,7 +253,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
         Friend Overrides Function GetImports() As ImmutableArray(Of String)
```

Comment:
Contract of this function is unclear. What is it supposed to produce? For example, if there is an alias in global imports, result will contain a string like "<alias> = <name>". Is this what expected?